### PR TITLE
Further extend basetest.pm code coverage

### PR DESCRIPTION
**Pull request by r-richardson**
WIP (continuation of: https://github.com/os-autoinst/os-autoinst/pull/2207)

see: https://progress.opensuse.org/issues/94952

### CODE REVIEW

File: t/17-basetest.t [hunk 1]
```diff
@@ -471,6 +484,24 @@ subtest verify_sound_image => sub {
     is_deeply($res->{area}, [{x => 1, y => 2, similarity => 100}], 'area was returned') or diag explain $res->{area};
     is($res->{needle}->{file}, 'foundneedle.json', 'needle file was returned');
     is($res->{needle}->{name}, 'foundneedle', 'needle name was returned');
+    # $suppress_match = 'yes';
+    # my $test_without_match = basetest->new();
+    # $res = $test_without_match->verify_sound_image("$FindBin::Bin/data/frame1.ppm", 'notapath2', 'check');
+    # diag explain $res->{result};
```
Comment (kalikiana) [inline t/17-basetest.t:490]: You may want to print `$res` here if result is empty or undefined

**Review by kalikiana — APPROVED** [review]
Looks good now I'd say; the CI failure is unrelated

**Review by Martchus — APPROVED** [review]

**Comment by Martchus** [comment]
Yes, the CI failure also happened on my PR. It is even before tests are starting (likely related to the package caching).

**Comment by kalikiana** [comment]
@mergifyio rebase

**Pull request merged**